1890rules-en-translation.tex

```diff
@@ -640,7 +640,7 @@ \subsubsection{Preparation for starting a public company}
 
 \subsection{Company closure}
 Private companies will be closed at the beginning of Phase 4
-(exceptions: Hankan Electric Railway, C8.4, D8.5: Osaka City Tramway).
+(exceptions: Hankai Tramway (see scenarios C8.1, D8.1): Osaka Municipal Electric Railway).
```

Comment:
While we're fixing this, let's make it a proper `\ref` (you may need to create a corresponding `\label` in the the scenario appendix)